packages/SwingSet/src/controller/initializeSwingset.js

```diff
@@ -233,15 +234,14 @@ async function normalizeConfigDescriptor(desc, referrer, expectParameters) {
  *    invalid.
  */
 export async function loadSwingsetConfigFile(configPath) {
+  await null;
```

Comment:
why does the rule error with the `try {}`? The first await is unconditional

Reply:
```js
try {
   await f();
} finally {
   g();
}
```
If `f()` throws, then `g()` happens in the same turn. If `f()` does not throw, then `g()` happens in the next turn.

Reply:
Even easier to miss:
```js
try {
  await f();
} catch (...) {
  ...
}
g();
```